diff --git a/node_modules/react-native/scripts/react_native_pods.rb b/node_modules/react-native/scripts/react_native_pods.rb
index be8b0ce..581dfa0 100644
--- a/node_modules/react-native/scripts/react_native_pods.rb
+++ b/node_modules/react-native/scripts/react_native_pods.rb
@@ -71,7 +71,7 @@ end
 # - config_file_dir: directory of the `package.json` file, required by the New Architecture.
 # - ios_folder: the folder where the iOS code base lives. For a template app, it is `ios`, the default. For RNTester, it is `.`.
 def use_react_native! (
-  path: "../node_modules/react-native",
+  path: "../../../node_modules/react-native",
   fabric_enabled: false,
   new_arch_enabled: NewArchitectureHelper.new_arch_enabled,
   production: false, # deprecated
@@ -282,7 +282,7 @@ end
 # - enable_hermes_profiler: whether the hermes profiler should be turned on in Release mode
 def react_native_post_install(
   installer,
-  react_native_path = "../node_modules/react-native",
+  react_native_path = "../../../node_modules/react-native",
   mac_catalyst_enabled: false
 )
   ReactNativePodsUtils.turn_off_resource_bundle_react_core(installer)
