diff --git a/node_modules/react-native-gesture-handler/android/build.gradle b/node_modules/react-native-gesture-handler/android/build.gradle
index 8afc3d5..76668f1 100644
--- a/node_modules/react-native-gesture-handler/android/build.gradle
+++ b/node_modules/react-native-gesture-handler/android/build.gradle
@@ -22,7 +22,7 @@ def safeExtGet(prop, fallback) {
 // Check whether Reanimated 2.3 or higher is installed alongside Gesture Handler
 def shouldUseCommonInterfaceFromReanimated() {
     if (rootProject.subprojects.find { it.name == 'react-native-reanimated' } != null) {
-        def inputFile = new File(rootProject.projectDir, '../node_modules/react-native-reanimated/package.json')
+        def inputFile = new File(rootProject.projectDir, '../../../node_modules/react-native-reanimated/package.json')
         def json = new JsonSlurper().parseText(inputFile.text)
         def reanimatedVersion = json.version as String
         def (major, minor, patch) = reanimatedVersion.tokenize('.')
