diff --git a/node_modules/react-native/react.gradle b/node_modules/react-native/react.gradle
index 5995ad5..ff8ae4d 100644
--- a/node_modules/react-native/react.gradle
+++ b/node_modules/react-native/react.gradle
@@ -33,15 +33,15 @@ def detectCliPath(config) {

     if (cliPath) {
         return cliPath
-    } else if (new File("${projectDir}/../../node_modules/react-native/cli.js").exists()) {
-        return "${projectDir}/../../node_modules/react-native/cli.js"
+    } else if (new File("${projectDir}/../../../../node_modules/react-native/cli.js").exists()) {
+        return "${projectDir}/../../../../node_modules/react-native/cli.js"
     } else {
         throw new Exception("Couldn't determine CLI location. " +
             "Please set `project.ext.react.cliPath` to the path of the react-native cli.js");
     }
 }
 
-def composeSourceMapsPath = config.composeSourceMapsPath ?: "node_modules/react-native/scripts/compose-source-maps.js"
+def composeSourceMapsPath = config.composeSourceMapsPath ?: "../../node_modules/react-native/scripts/compose-source-maps.js"
 def bundleAssetName = config.bundleAssetName ?: "index.android.bundle"
 def entryFile = detectEntryFile(config)
 def bundleCommand = config.bundleCommand ?: "bundle"
@@ -49,7 +49,7 @@ def reactRoot = file(config.root ?: "../../")
 def inputExcludes = config.inputExcludes ?: ["android/**", "ios/**"]
 def bundleConfig = config.bundleConfig ? "${reactRoot}/${config.bundleConfig}" : null ;
 def enableVmCleanup = config.enableVmCleanup == null ? true : config.enableVmCleanup
-def hermesCommand = config.hermesCommand ?: "../../node_modules/hermes-engine/%OS-BIN%/hermesc"
+def hermesCommand = config.hermesCommand ?: "../../../../node_modules/hermes-engine/%OS-BIN%/hermesc"

 def reactNativeDevServerPort() {
     def value = project.getProperties().get("reactNativeDevServerPort")
